kernel/src/main/scala/cats/kernel/std/map.scala

```diff
@@ -1,16 +1,11 @@
 package cats.kernel
 package std
 
-import cats.kernel.std.util.StaticMethods.{ addMap, subtractMap }
+import cats.kernel.std.util.StaticMethods.addMap
 
 package object map extends MapInstances
 
-trait MapInstances extends MapInstances0 {
-  implicit def mapGroup[K, V: Group]: MapGroup[K, V] =
-    new MapGroup[K, V]
-}
-
-trait MapInstances0 {
+trait MapInstances {
```

Comment:
I'll add that optimization to the other instances that would benefit (e.g. `String`, `Vector`, `List`, `Stream`, etc).